internal/runners/packages/remove.go

```diff
@@ -33,23 +32,12 @@ func NewRemove(prime primeable) *Remove {
 
 // Run executes the remove behavior.
 func (r *Remove) Run(params RemoveRunParams) error {
+	logging.Debug("ExecuteRemove")
 	err := r.run(params)
 	headless.Notify(r.out, r.proj, err, "packages")
 	return err
 }
 
 func (r *Remove) run(params RemoveRunParams) error {
-	fail := auth.RequireAuthentication(locale.T("auth_required_activate"), r.out, r.Prompter)
-	if fail != nil {
-		return fail.WithDescription("err_activate_auth_required").ToError()
-	}
-
-	// Commit the package
-	pj := project.Get()
-	language, fail := model.DefaultLanguageNameForProject(pj.Owner(), pj.Name())
-	if fail != nil {
-		return locale.WrapError(fail, "err_fetch_languages")
-	}
-
-	return executePackageOperation(r.out, r.Prompter, language, params.Name, "", model.OperationRemoved)
+	return executePackageOperation(r.out, r.Prompter, "", params.Name, "", model.OperationRemoved)
```

Comment:
Language can't be empty.. ?

Reply:
For remove, we do not need to know the language.